wallet/pskt/src/global.rs

```diff
@@ -39,6 +39,7 @@ pub struct Global {
     /// Unknown key-value pairs for this output.
     #[serde(flatten)]
     pub unknowns: BTreeMap<String, serde_value::Value>,
+    pub payload: Option<Vec<u8>>,
```

Comment:
Add attribute `#[serde(with = "kaspa_utils::serde_bytes_optional")]` to serialize it as hex in case of human readable serializer, as well as it has consistency with redeem script serialization

Reply:
Thanks very much for review I'll address soon

Reply:
Hi @biryukovmaxim I have updated the pr